app/models/story.rb

```diff
@@ -212,6 +214,19 @@ def v4?
     app_version == PRX::APP_VERSION
   end
 
+  # for each distribution, create a story distribution
+  def create_story_distributions
+    return unless series
+    series.distributions.each do |distro|
+      begin
+        distro.create_story_distribution(self)
+      rescue StandardError => err
+        logger.error(err)
+        NewRelic::Agent.notice_error(err)
```

Comment:
yes, we can expand using this - copying what you did in dovetail. I think if we want to do this more in the rails apps, we should probably abstract it, on the off chance that we quit using new relic, or so we can easily stub or change all error noticing